Makefile

```diff
@@ -158,3 +158,21 @@ verify: api-verify backend-verify frontend-verify
 .PHONY: yarn-ensure # Install the pinned version of yarn.
 yarn-ensure:
 	@./tools/install-yarn.sh
+
+.PHONY: k8s-start # Start a local k8s cluster
+k8s-start:
```

Comment:
let's prefix this with `dev-`, we can later have other targets like `dev-db-up` `dev-db-down` and a general `dev-up`, `dev-down` that calls the other dev env targets

edit: also let's call this `up` instead of `start` just to keep things consistent between docker compose and our make targets